### PR TITLE
supported JRE/JDK should be not 1.7 but 1.8

### PR DESCRIPTION
**Pull request by KengoTODA**
It seems that first SpotBugs release [doesn't support 1.7](https://github.com/spotbugs/spotbugs/issues/19), then it's better to update this page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 <p>SpotBugs is the spiritual successor of <a href="http://findbugs.sourceforge.net/">FindBugs</a>, carrying on from the point where it left off with support of its community.</p>
 
-<p>SpotBugs requires JRE (or JDK) 1.7.0 or later to run.  However, it can analyze programs compiled for any version of Java, from 1.0 to 1.8.</p>
+<p>SpotBugs requires JRE (or JDK) 1.8.0 or later to run.  However, it can analyze programs compiled for any version of Java, from 1.0 to 1.8.</p>
```
Comment (jsotuyod) [inline index.html]: we could take the chance to say it can analyze any java program from 1.0 to 1.9 actually

Comment (KengoTODA) [reply index.html]: Good, I will update in this branch.

**Comment by jsotuyod** [comment]
Amazing! Thanks! (BTW, I have sent you an invite to join the SpotBugs team)

**Comment by KengoTODA** [comment]
Oh, I found invitation, thanks.

**Pull request merged**